includes/helper/class-wp-job-manager-helper-api.php

```diff
@@ -67,44 +67,70 @@ public function plugin_information( $args ) {
 	/**
 	 * Attempt to activate a plugin licence.
 	 *
-	 * @param array|string $args
-	 * @return boolean|string JSON response or false if failed.
+	 * @param array $args The arguments to pass to the API.
+	 * @return array|false JSON response or false if failed.
 	 */
 	public function activate( $args ) {
-		$args            = wp_parse_args( $args );
-		$args['wc-api']  = 'wp_plugin_licencing_activation_api';
-		$args['request'] = 'activate';
-		$response        = $this->request( $args, true );
-		if ( false === $response ) {
+		$args         = wp_parse_args( $args );
+		$product_slug = $args['api_product_id'];
+		$response     = $this->bulk_activate( $args['licence_key'], [ $product_slug ] );
+		if ( false === $response || ! array_key_exists( $product_slug, $response ) ) {
 			return false;
 		}
-		return $response;
+		$item = $response[ $product_slug ];
+		// Add keys used previously for backwards compatibility.
+		$item['activated'] = $item['success'];
+		unset( $item['success'] );
+		if ( ! $item['activated'] ) {
+			$item['error'] = $item['error_message'];
+			unset( $item['error_message'] );
+		}
```

Comment:
Another option could be to directly map the new structure to the old one. Instead of adding/removing the activated/success/error/error_message items.

Reply:
Good one! Done in 33c20ac, thanks.